test/e2e/common.go

```diff
@@ -192,6 +192,37 @@ func DownloadFromAppInWorkloadCluster(ctx context.Context, workloadKubeconfigPat
 	return KubectlExec(ctx, "run", workloadKubeconfigPath, runArgs...)
 }
 
+func DownloadMetricsFromCAPCManager(ctx context.Context, bootstrapKubeconfigPath string) (string, error) {
+	// Expose the CAPC manager metrics port via a K8S service
+	runArgs := []string{
+		"--port=8080", "--target-port=metrics", "--name=capc-controller-manager-metrics", "--namespace=capc-system", "deployment", "capc-controller-manager",
+	}
+	_, err := KubectlExec(ctx, "expose", bootstrapKubeconfigPath, runArgs...)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Remove if this is just for debugging

Reply:
Either of
``` go
			Ω(KubectlExec(ctx, "expose", bootstrapKubeconfigPath, runArgs...)).Should(Succeed())
```
``` go
			_, err := KubectlExec(ctx, "expose", bootstrapKubeconfigPath, runArgs...)
			Ω(err).Error().ShouldNot(HaveOccurred())
```
Should work nicely if ginkgo/gomega is in use, and I was pretty sure it was in the E2Es.

Reply:
Yeah, it's imported, so as long as it's setup right, you should be able to drop in either of the above snippets.

Reply:
I went with the second form.  In the first form Ω seemed to mix up the result and error return params.  Also, I needed the second form for the KubeCtl exec where I capture the response so I the func can return it.